lib/metasploit/framework/aws/client.rb

```diff
@@ -0,0 +1,175 @@
+
+require 'msf/core'
+require 'net/http'
```

Comment:
Are you using anything from this?  It seems like you are just using `Msf::Exploit::Remote::HttpClient`